### DISCUSSION TITLE
Open CmdLine in the bottom (default vim) instead of floating window

### DISCUSSION THREAD

**Discussion by CaptainJack42** [LazyVim/LazyVim#337] [Q&A] [answered]
Hey, i've been searching for the plugin/setting that is causing the CmdLine opening in a floating window in the middle of the screen for a while now, but can't seem to find it. I'd like to disable this behavior, but as mentioned can't find the setting.

Any help is appreciated,

cheers!

**Comment by loiccoyle** [comment, marked as answer]
Look at the noice wiki: https://github.com/folke/noice.nvim/wiki/Configuration-Recipes#classic-cmdline

**Reply by CaptainJack42** [reply]
That worked, thx a lot!
